docs/src/main/asciidoc/se/openapi/openapi-generator.adoc

```diff
@@ -23,9 +23,282 @@
 :rootdir: {docdir}/../..
 
 include::{rootdir}/includes/se.adoc[]
+:incdir: {rootdir}/includes/openapi
```

Comment:
Current pattern used:

- 1:1 between `.adoc` and `.java`, one java class suffixed with `Snippets` per `.adoc` file
  * static nested classes / static imports are used as "namespace" tricks
- snippets are named `snippet_[id]` where id is the snippet number starting at 1
- includes are done like this:
   ```adoc
   include::{sourcedir}/mp/RestclientSnippets.java[tag=snippet_1, indent=0]
   ```

Reply:
Yes, I'm aware of how to stub whatever's needed by a single file in that file.

I had kept separate files where they are referenced from multiple places or where it seemed important to be more faithful to the original class but I get that this has not been a priority so far.

I'll make further changes to reduce the file count some more.